roles/common/files/sensu_plugins/check-glance-store.py

```diff
@@ -1,16 +1,19 @@
 #!/usr/bin/env python2.7
 
-from datetime import datetime, timedelta
-from glanceclient import client
-from iso8601 import iso8601
-from keystoneclient.openstack.common import timeutils
```

Comment:
timeutils is no longer part of keystoneclient.openstack.common.  Hence a bit change, same logic.